docs/install.rst

```diff
@@ -36,3 +36,41 @@ Run the following commands to setup the database tables::
   $ python manage.py check --deploy
 Finally, restart the python process running modoboa (uwsgi, gunicorn,
 apache, whatever).
+
+Uninstallation
+------------
+
+Edit the settings.py file of your modoboa instance and comment the line with
+``modoboa_amavis`` inside the ``MODOBOA_APPS`` variable like this::
+
+    MODOBOA_APPS = (
+        'modoboa',
+        'modoboa.core',
+        'modoboa.lib',
+        'modoboa.admin',
+        'modoboa.relaydomains',
+        'modoboa.limits',
+        'modoboa.parameters',
+        # Extensions here
+        # ...
+        # 'modoboa_amavis',
+    )
+
+Then, comment the following at the end of the file::
+
+  # from modoboa_amavis import settings as modoboa_amavis_settings
+  # modoboa_amavis_settings.apply(globals())
+
+Run the following commands to setup the database tables::
+
+  $ cd <modoboa_instance_dir>
+  $ python manage.py migrate
```

Comment:
This won't have any effect...

Reply:
Can you elaborate that a little? I am uncertain what manage.py migrate does

Reply:
@H-A-L-I-S you have a guide in the Django documentation (remove app on settings.py is not enougth)
see https://docs.djangoproject.com/en/5.0/howto/delete-app/